plugins/speculation-rules/class-plsr-url-pattern-prefixer.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Class 'PLSR_URL_Pattern_Prefixer'.
+ *
+ * @package speculation-rules
+ * @since n.e.x.t
```

Comment:
What about version? When we change it from `n.e.x.t` to actual version `1.0.0`

Reply:
Since the plugin is already published, this should be already 1.0.0, no?